plugins/crane.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: crane
+spec:
+  homepage: https://github.com/gocrane/kubectl-crane
+  shortDescription: "Easily interact with crane"
```

Comment:
Is it possible to be a bit more descriptive for someone who's not familiar.
At least this should probably capitalize C in Crane?